lib/storekit/receipt_response.rb

```diff
@@ -0,0 +1,27 @@
+module StoreKit
+  class ReceiptResponse
```

Comment:
I am not sure what is driving these changes, but I don't think we need to rename classes and break existing behaviour.